src/components/FeatureNotification/index.tsx

```diff
@@ -0,0 +1,106 @@
+import UserNotification from './UserNotification';
+import { Notification } from 'src/types/Notifications';
+import { useEffect, useState } from 'react';
+import isEnabled from 'src/features';
+import { useOrganization, useTimeZones, useUser } from 'src/providers';
+import { InitializedTimeZone, TimeZoneDescription } from 'src/types/TimeZones';
+import { getTimeZone, getUTC } from 'src/utils/useTimeZoneUtils';
+import { OrganizationService, UserService } from 'src/services';
+import { InitializedUnits } from 'src/units';
+import OrganizationNotification from './OrganizationNotification';
+
+export default function useFeatureNotifications(): Notification[] {
+  const [unitNotification, setUnitNotification] = useState(false);
```

Comment:
this component should ideally only import the various feature notifications and return the list of those feature notifications.
maybe this component can also have a 'markAll' property which it can pass down to feature notificaitions.. if it's true the individual notifications can mark it as read/unread etc..if needed..

just some thoughts..

basically .. having all the feature logic in individual components will be more scalable